src/fcntl.rs

```diff
@@ -549,6 +551,11 @@ pub fn fcntl(fd: RawFd, arg: FcntlArg) -> Result<c_int> {
             F_GETPIPE_SZ => libc::fcntl(fd, libc::F_GETPIPE_SZ),
             #[cfg(any(target_os = "linux", target_os = "android"))]
             F_SETPIPE_SZ(size) => libc::fcntl(fd, libc::F_SETPIPE_SZ, size),
+            #[cfg(any(target_os = "netbsd", target_os = "macos", target_os = "ios"))]
+            F_GETPATH(path) => {
+                path.resize(libc::PATH_MAX);
```

Comment:
`Vec::resize()` takes two arguments, this is the reason why [CI](https://github.com/nix-rust/nix/pull/2142/checks?check_run_id=17282030491) failed